test/e2e/apps/replica_set.go

```diff
@@ -35,7 +36,37 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-func newRS(rsName string, replicas int32, rsPodLabels map[string]string, imageName string, image string) *extensions.ReplicaSet {
+func newRS(rsName string, replicas int32, rsPodLabels map[string]string, imageName string, image string) *apps.ReplicaSet {
```

Comment:
a 1.7 cluster will definitely not have apps/v1beta2 replicasets, right? @kubernetes/sig-testing-pr-reviews, am I correct that version skew tests can run e2e tests from master against prior cluster versions? I'm assuming the answer is yes based on the presence of things like https://github.com/kubernetes/kubernetes/blob/1957e4a2ced19d0beb85da9c232b8eb7ce2e771b/test/e2e/auth/service_accounts.go#L246